documents_stock_production_lot/models/documents_workflow_rule.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2021 Open Source Integrators
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class DocumentsWorkflowRule(models.Model):
+    _inherit = ["documents.workflow.rule"]
+
+    def _compute_business(self):
+        return self._get_business()
+
+    has_business_option = fields.Boolean(default=True, compute="_compute_business")
```

Comment:
```suggestion
    has_business_option = fields.Boolean(default=True, compute="_get_business")
```

Reply:
@bodedra Compute methods should start with `_compute`

Reply:
@max3903 that's true. I added suggestion based on Odoo Enterprise implementation on the documents modules. ref: documents_****